application/src/test/java/org/opentripplanner/street/model/edge/StreetEdgeGeofencingTest.java

```diff
@@ -294,6 +300,56 @@ private static StreetSearchRequest defaultArriveByRequest() {
         .withArriveBy(true)
         .build();
     }
+
+    @Test
+    public void hslFailTst() {
```

Comment:
Can you give this tests a name that explains what it tests?

Reply:
Also, a small thing, the test methods don't need to be public.